views/allbooks.handlebars

```diff
@@ -0,0 +1,25 @@
+<body>
+<h1>These are all books that are available on BookSwap</h1>
```

Comment:
This page should mimic the look and feel of the my library page. The results should not be in a table but should be cards that will wrap down the page. I'll approve it like this for now but wanted to add this comment so we don't forget this needs to be updated.